cpp/src/barretenberg/honk/composer/composer_helper/standard_honk_composer_helper.hpp

```diff
@@ -19,14 +19,17 @@ class StandardHonkComposerHelper {
     using CircuitConstructor = Flavor::CircuitConstructor;
     using ProvingKey = Flavor::ProvingKey;
     using VerificationKey = Flavor::VerificationKey;
+    using PCSCommitmentKey = PCSParams::CommitmentKey;
 
     static constexpr size_t NUM_RESERVED_GATES = 2; // equal to the number of multilinear evaluations leaked
     static constexpr size_t NUM_WIRES = CircuitConstructor::NUM_WIRES;
     std::shared_ptr<ProvingKey> proving_key;
     std::shared_ptr<VerificationKey> verification_key;
-    // TODO(#218)(kesha): we need to put this into the commitment key, so that the composer doesn't have to handle srs
-    // at all
+    // The crs_factory holds the path to the srs and exposes methods to extract either the prover srs (collection of g1
+    // points) or verifier srs ([x]_2)
```

Comment:
this is very kzg specific, maybe don't mention what the prover and verifier srs contain?